_posts/2023-05-19-story.md

```diff
@@ -0,0 +1,608 @@
+_I'm sorry sir. We only wanted to investigate it._
+
+A man in a combat uniform entered. The patch on his shoulder showed the triangular Space Force logo. The gold rectangle patch on his chest indicated Second Lieutenant. 
+
+"Eva. Remember me?"
+"Yeah, Eckert. You work with Bill, right?"
+"Let's talk." He motioned for her to follow. 
+
+_Here it comes_.
+
+They placed their phone and keys in lockers, walked through metal detectors, then through a thick steel doorway doorway. The windowless room was the size of an average conference room, with walls made to prevent any kind of electronic eavesdropping.
+
+The man swung the heavy door closed behind him. 
+
+"I almost forgot we had one of these," she said. 
+
+He didn't seem interested in smalltalk. 
+
+"I apologize that Bill couldn't be here in person to deliver the news himself. I'll cut to the chase. What I'm about to tell you is classified."
+
+She nodded. 
+
+"As you probably know, tensions with the EAC were already high after Flight 766. Now it's worse. Two of their civilian aircraft crashed and we still don't know the status of the victims. We have high confidence it was because GPS spoofing from the electronic warfare attack caused their autopilot to malfunction. They're blaming us for the attack."
+
+He continued. 
+
+"And now for the classified part, which is why I brought you here. We have a pretty good idea of the EAC's and other adversaries' capabilities. We know our capabilities. Whatever did this is beyond *anyone's* capabilities."
+
+"In what way?" she asked. 
+
+"The electronic warfare attack, while impressive, is within the realm of possibility. But it's not _how_ they did the attack we're interested in. It's _where_ the attack came from. It essentially came out of nowhere and shows no resemblance to any known satellite's orbit, including those of our adversaries."
+
+"And what did the EAC say about that?"
+
+"Our backchannels in the EAC said cooler heads have not prevailed. Their regime is under intense pressure for legitimacy and they might lash out to score some political points. They're moving destroyers with anti-satellite missiles into offensive positions."
+
+He continued. 
+
+"That's where you come in. That _Sunfly_ rocket to LEO Lodge? It's getting repurposed to visit the unknown object. You're going to rendezvous with it and get whatever information you can. It asked for you so that's who we're sending."
+
+Eva felt uneasy. She wasn't sure whether to be excited or scared. 
+
+"Alright. Great! Who's on my crew?"
+
+"You will be accompanied by extra hydrazine."
+
+"So, more fuel."
+
+"The object is in a higher orbit than LEO Lodge and the current configuration won't get that far. Replacing some crew with fuel should get us where we need. Any questions?"
+
+"I want Talia on comms. She knows this thing better than anyone else."
+
+"Talia Wescoff? I think we can arrange that."
+
+<!-- Launch -->
+
+Eva looked at the black sky through the crew capsule. 
```

Comment:
Are you planning to make these visible sub-headers between sections, and/or are you planning to fill out these scene transitions more? I like that they err on the side of stopping once they've advanced the story as far as you need them to, rather than being padded out, but the first few are abrupt. Maybe try extending them a bit based on the emotion or character motivation you want each scene to present. Eva meets with Eckert, expecting him to punish her but instead finds out she's been tagged for a solo space mission. How is she feeling when she exits this conversation -- afraid, excited, unreal? Is there a tangible thing she would do on her way out of this scene -- leaving the meeting room, walking outside, arriving back at her home -- that would show some of what's going on inside her?

Reply:
Originally I had them as subheaders but it broke the flow. They're just for my own uses.